srcs/utils/StringUtils.cpp

```
@@ -13,8 +13,18 @@ std::string trim(const std::string &str)
 
 void remove_content_from_packet(std::string &packet, const std::string &content)
 {
-	size_t content_start = packet.find(content);
+	const size_t content_start = packet.find(content);
 	if (content_start == std::string::npos)
 		return;
 	packet.erase(content_start, content.size());
 }
+
+std::string &toLowerCase(std::string &str)
```

cppcheck

[style] The function 'toLowerCase' is never used.

(unusedFunction)